src/Komponents/TittleColection/Tittle.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Tittle = ({value ,style}) => {
```

Comment:
u can skip brackets here for Title component. Because u don't create any logic here and it it arrow function